apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -153,10 +157,13 @@ export default function RowColumn({ index, style, data, column = false }) {
     checkboxes = false,
     dense = false,
     frequencyMode = 'N',
+    actions,
     frequencyMax = '',
     histogram = false,
   } = data;
 
+  const handleKeyDownCallback = useCallback(getKeyboardNavigation(actions), [actions]);
```

Comment:
I think the useCallback here will have very limited effect as the the parent component send in a new actions object every times it re-render. (It still could help for the case when only this and not the parent component is updated)

Reply:
Yeah agreed. I wrap this mostly as a best practice/habit than intentional optimisation.